l10n/bundle.l10n.zh-cn.json

```diff
@@ -32,11 +32,14 @@
   "Enter the URL": "输入网址",
   "Enter the URL for the SAS Viya server. An example is https://example.sas.com.": "输入SAS Viya服务器的URL。示例为 https://example.sas.com",
   "Enter the name of the SAS 9 SSH server.": "输入SAS 9 SSH服务器的名称。",
```

Comment:
Hi, it would be great to remove unused messages

Reply:
Thanks for the reminder :)

The locale tool now cleans these up, introduced in https://github.com/sassoftware/vscode-sas-extension/pull/793/commits/b6be8d8711d6e0e62d44300ad738e5fe19a6e89b.

The results of that are in https://github.com/sassoftware/vscode-sas-extension/pull/793/commits/df69c736a6f8486766f938cad121dc3a8dca0b47

Reply:
Made a couple more updates to do the same for package nls files - https://github.com/sassoftware/vscode-sas-extension/pull/793/commits/d0e2eeaef76bdfc7190285346745ea45872a676b / https://github.com/sassoftware/vscode-sas-extension/pull/793/commits/0e84ac7323c5ef07370a77bc1bedf71847faee03